rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-builder-6.4.0-release-notes.adoc

```diff
@@ -1,24 +1,35 @@
 = RPA Builder 6.4.0 Release Notes
 
-RPA Builder is the on-premise application that enables you to build automations by specifying the action steps of the process.
+RPA Builder is the on-premises application that enables you to build automations by specifying the action steps of the process.
 
 include::partial$release-notes/rn-last-version-message.adoc[] 
 
-//TODO
-== July TBD, 2023
+== July 31, 2023
 
 // == What's New
 include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
 
-//
+* The new JSONPath Expression Builder enables users to build JSONPath expressions by importing a JSON file and selecting an element in the UI.
++
+JSONPath Expression Builder is available in the wizards for Action Steps that support JSONPath expressions and in the menu bar of RPA Builder. 
+* The new Save Excel File Action Step enables users to save the changes to the Excel file being modified inside the Excel Session at any point of the execution.
+* RPA Builder now creates Mouse Action and Search Pattern Action Steps based on a recording from RPA Recorder. 
+* Excel Session performance is improved and now supports: 
+** Additional file formats (`XLSM`, `XLSB`, and `XLS`).
+** Processing files with more than 20.000 rows. 
+** Processing files with more thn 78 columns.
+** Opening and creating password-protected Excel files. 
+** Creating a new file the *Read and Write to the same file* operation mode. 
```

Comment:
Just flagging this to see if "file the" should be removed from the beginning of this sentence? Should it read: 

Creating a new *Read and Write to the same file* operation mode.